data_science/datasciencetopics.md

```diff
@@ -0,0 +1,66 @@
+# Data Science Topics and Subtopics
+
+### Terminal
+- Directories
+- Terminal commands
+- Installation and setup
+- Jupyter notebook
+
+### Git and GitHub
+- Cloning Repositories
+- Version Control
+- Branching & merging
+- Pull requests & working with a team
+
+### Markdown
+- Intro to basic syntax (i.e. headings, lists, code blocks, embedding links and images)
+
+### Progrmming Concepts
+- Iteration
+- Conditional Statements
+- Sequences (Array, List)
+- Data Types (Strings, numbers, logical)
+
+### Python 1 - Intro to Python
+- Integrating programming concepts
+- Dictionaries
+- Lists
+- Tables
+- Operational and Logical Operands
+
+### Python 2 - Functions
+- Writing functions
+- Try-catch
+
+### Pandas and Seaborn
+- Dataframes ( creation i.e. importing and starting from scratch, manipulation, merging, etc.)
+- Graph types and capabilities
+- Graph properties (i.e. data point size, colour, x and y labels, title, subheading, font size)
+
+### Excel
+- Excel commands and functions
+
+### Tableau
+- Tableau capabilities
+- Creating effective dashboards 
+
+### Data Visualization and Data Analysis
+- Exploratory Data Analysis
+- Graph Types and uses for each one
+
+### Statistics and Probability
+- Estimation and Confidence Intervals
+- Simulations
+- Distribution (i.e. standard, normal)
+- Central Limit Theorem
+- Standard Deviation
+- Sampling
+- Randomness
+- Prediction

```

Comment:
I've usually seen prediction separated by regression and classification, so I was wondering if making it less general would be useful.